test/data/mock.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * Keep in sync with /test/middleware-mockserver.js
+ * Keep in sync with /test/middleware-mockserver.cjs
```

Comment:
BTW, `mock.php` on `main` has some differences from the `3.x-stable` version, mostly due to response formatting (3.x has more spaces). Spacey versions better suite our code style, but we should align regardless. `middleware-mockserver.cjs` has way fewer changes - mostly extra stuff for `main` which makes sense.

Reply:
I've copied what changes I thought made sense from main, but not the spacing. We can then switch the spacing on main.